.github/workflows/js_tests.yml

```diff
@@ -1,6 +1,6 @@
 name: JS Tests
 
-on: [push, pull_request, workflow_dispatch]
```

Comment:
Removing `pull_request` since `push` covers our use cases.

`pull_request`: 
> The action that was performed. Can be one of opened, edited, closed, assigned, unassigned, review_requested, review_request_removed, ready_for_review, converted_to_draft, labeled, unlabeled, synchronize, auto_merge_enabled, auto_merge_disabled, locked, unlocked, or reopened. If the action is closed and the merged key is false, the pull request was closed with unmerged commits. If the action is closed and the merged key is true, the pull request was merged.

ref: https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#pull_request

`push`: 
> One or more commits are pushed to a repository branch or tag.

ref: https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#push